textarea/elfinder-sakai/src/java/org/sakaiproject/elfinder/sakai/samigo/SamSiteVolumeFactory.java

```diff
@@ -34,9 +34,7 @@
 
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
I know star imports are not terribly "bad", but I think we shouldn't use them...